watertap/property_models/unit_specific/cryst_prop_pack.py

```diff
@@ -797,6 +797,45 @@ def build(self):
             doc="Solution enthalpy parameter E5",
         )
 
+        # Specific volume parameters, computed from Affandi expression (Eq 5)
```

Comment:
@kurbansitterley can you some description for this in the associated documentation? And include this reference?